sakai-evaluation-tool/src/webapp/content/templates/modify_hierarchy_node_perms.html

```diff
@@ -59,10 +59,18 @@ <h3 rsf:id="msg=modifynodeperms.page.title">Hierarchy Node Perms</h3>
 			Page Instructions: Enter usernames or emails in the box and then select the permissions to apply to the users
 		</p>
 		
-		<form rsf:id="perms-form" method="post">
+		<form rsf:id="perms-form" method="post" style="display: inline-block;">
 			
 			<input type="hidden" rsf:id="node-id" />
-			
+
+			<!-- Clear changes and return to hierarchy UI buttons at the top of the form -->
+			<div class="action" style="margin-bottom: 5px;">
+				<span class="cancelButton">
+					<a href="#" rsf:id="return-link1">Return to Control Hierarchy</a>
+					<input type="submit" rsf:id="cancel-changes-button1" style="float: right;" />
```

Comment:
Same as above comment; should likely be addressed in a separate JIRA project wide.